index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Video App</title>
+  <link rel="stylesheet" href="./index.css" />
+</head>
+
+<body>
+
+  <div class="main">
+
+    <nav class="navbar">
+      <div class="brand">
+        <img src="./img/menu.png" alt="menu-icon" />
+        <span>Video App</span>
+      </div>
+
+      <div class="input-field">
+        <input class="searchBar" type="text" />
+        <img src="./img/search.png" alt="search-icon" />
+      </div>
+
+      <div class="menu-section">
+        <ul class="top-menu">
+
+          <li><a href=""><img src="./img/notification.png" alt="" /></a></li>
+          <li><a href=""><img src="./img/video-camera.png" alt="" /></a></li>
+          <li><a href=""><img class="avatar" src="https://source.unsplash.com/random" alt="" /></a>
+          </li>
+
+        </ul>
+
+      </div>
```

Comment:
Please remove these whitespaces between your codes throughout this file because they make your code difficult to read and not clean 😃